python-ecosys/aiohttp/aiohttp/__init__.py

```diff
@@ -38,10 +38,10 @@ async def read(self, sz=-1):
         return self._decode(await self.content.read(sz))
 
     async def text(self, encoding="utf-8"):
-        return (await self.read(sz=-1)).decode(encoding)
+        return (await self.read(sz=int(self.headers.get("Content-Length", -1)))).decode(encoding)
```

Comment:
I suggest just using positional arguments (not doing `sz=`) because it will make slightly smaller and slightly faster code.